src/processors/knex-processor.ts

```diff
@@ -255,12 +255,12 @@ export default class KnexProcessor<
       return query
         .join(
           this.tableName,
-          `${foreignTableName}.id`,
+          `base${foreignTableName}.id`,
```

Comment:
@spersico What does `base` mean? I know it works as a prefix, but I'd like to know why do we call it that way.

Reply:
because.... I felt like it :-P

Reply:
I'll give it a more descriptive name, like... **other** or something like that, I'll check the generated sql string and see